swigibpy.py

```diff
@@ -2036,7 +2036,10 @@ def run(self):
         pollerr = [fd]
 
         while self._tws and self._tws.isConnected():
-            evts = select(pollin, pollout, pollerr)
+            try:
+                evts = select(pollin, pollout, pollerr)
+            except OSError:
+                break
```

Comment:
Put the rest of the loop in an `else` clause as `evts`  is only defined in scope of the `try` clause. 

i.e. do

```
try:
    ...
except:
    ....
else:
    if fd in evts[0]:
        ...
    else:
        ...
```